### PR TITLE
21 dashboard

### PR DESCRIPTION
**Pull request by Djesper826**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,19 +1,18 @@
-version: "3.8"
-
 services:
   db:
     image: mysql:8.0
-    env_file:
```
Comment (MartAtSiip) [inline docker-compose.yml:6]: why did you remove this?

File: src/views/dashboard.php [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+// Database credentials, very secure lol
+$host = "192.168.1.11";
+$dbname = "thesixthstring";
+$username = "default";
+$password = "rEN28Sd8?W|L6FquVky>";
+
+try {
+    // Database connection
+    $dsn = "mysql:host=$host;dbname=$dbname;charset=utf8mb4";
+    $pdo = new PDO($dsn, $username, $password);
```
Comment (MartAtSiip) [inline src/views/dashboard.php:20]: please use db.php for connection

**Comment by Djesper826** [comment]
fixed errors

**Review by martvdm — APPROVED** [review]

**Review by MartAtSiip — APPROVED** [review]
